core/ledger/kvledger/txmgmt/statedb/statecouchdb/couchdb.go

```diff
@@ -232,73 +231,47 @@ func (dbclient *couchDatabase) createDatabaseIfNotExist() error {
 		}
 	}
 
-	//If the dbInfo returns populated and status code is 200, then the database exists
-	if dbInfo != nil && couchDBReturn.StatusCode == 200 {
+	if dbInfo == nil || couchDBReturn.StatusCode != 200 {
```

Comment:
Since you inverted the condition, I find that explicitly checking `dbInfo == nil && couchDBReturn.StatusCode == 404` makes it more readable.

Just to take a note, the both parts of the condition are actually same but it seems that they were always like that.

Reply:
Done